samples/nrf_desktop/src/modules/ble_adv.c

```diff
@@ -74,6 +66,23 @@ static const struct bt_data ad[] = {
 #endif
 };
 
+
+enum state {
+	STATE_DISABLED,
+	STATE_IDLE,
+	STATE_ACTIVE_FAST,
+	STATE_ACTIVE_SLOW,
+	STATE_ACTIVE_FAST_DIRECT,
+	STATE_ACTIVE_SLOW_DIRECT,
+	STATE_GRACE_PERIOD
+};
+
+struct bond_scan_data {
```

Comment:
Can we name this structure `bonded_peer_data` or `peer_data`? It is not connected to BLE scanning, so better to avoid `scan` in the name.

Reply:
`bond_find_data` it is